doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond within 1-2 seconds.
+6. Should be user-friendly.
```

Comment:
user-friendly is not specific. For example, should be able to be operated by a new user within 15 minutes